### PR TITLE
Feat/revise UI

### PR DESCRIPTION
**Pull request by pitb2022**

### CODE REVIEW

File: src/components/Buttons/ViewMoreCard/index.tsx [hunk 1]
```diff
@@ -33,7 +33,13 @@ export const ViewMoreCard: React.FC<
 
   return (
     <section className={viewMoreCardClasses}>
-      <Card spacing={['base', 'base']} {...cardProps}>
```
Comment (robertu7) [inline src/components/Buttons/ViewMoreCard/index.tsx:36]: image

wrong spacing between button and card below?

File: src/views/Circle/Works/Works.tsx [hunk 2]
```diff
@@ -158,14 +159,6 @@ const CircleDetailWorks = () => {
                       id: node.id,
                     })
                   }
-                  onClickAuthor={() => {
```
Comment (robertu7) [inline src/views/Circle/Works/Works.tsx:161]: hmm, why remove this?

File: src/views/Me/Wallet/index.tsx [hunk 3]
```diff
@@ -123,7 +123,12 @@ const Wallet = () => {
 
       <ResponsiveWrapper>
         <Form.List spacingX={0}>
-          {hasPaymentPassword && <PaymentPassword />}
+          {hasPaymentPassword && (
+            <>
+              <PaymentPassword />
+              <hr className="dashed-line" />
```
Comment (robertu7) [inline src/views/Me/Wallet/index.tsx:129]: I didn't get why we need to add the `<hr>` ?

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: src/views/Circle/Works/Works.tsx (hunk 2 above)
Comment (pitb2022) [reply src/views/Circle/Works/Works.tsx:161]: We had hide author of the article card of the circle

**Review by robertu7 — APPROVED** [review]

**Pull request merged**